checkov/terraform/runner.py

```diff
@@ -61,6 +61,7 @@
 dpath.options.ALLOW_EMPTY_STRING_KEYS = True
 
 CHECK_BLOCK_TYPES = frozenset(['resource', 'data', 'provider', 'module'])
+ENABLE_DEFINITION_KEY = strtobool(os.getenv('ENABLE_DEFINITION_KEY', 'False'))
```

Comment:
I'm not sure about it - but if we will do the import of this file before creating this env var it might not work